charts/seed-monitoring/charts/prometheus/optional-rules/elasticsearch.rules.yaml

```diff
@@ -0,0 +1,14 @@
+groups:
+- name: elasticsearch.rules
+  rules:
+  - alert: ElasticsearchDown
+    expr: absent(up{job="elasticsearch-logging"} == 1)
+    for: 20m
+    labels:
+      service: elasticsearch-logging
+      severity: warning
+      type: seed
+      visibility: operator
+    annotations:
+      description: Elasticsearch could not be found. Logs will not be able to collect
```

Comment:
Change description to: There are no running elasticsearch pods. No logs will be collected.